TaskBuilder.fs.fsproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard1.6</TargetFramework>
+    <PackageVersion>1.0.0</PackageVersion>
+    <PackageId>TaskBuilder.fs</PackageId>
+    <Title>TaskBuilder.fs</Title>
+    <Authors>ForNeVeR</Authors>
```

Comment:
Please note that I'm not claiming authorship of the code, but NuGet spec [tells me](https://docs.microsoft.com/en-us/dotnet/core/tools/csproj#nuget-metadata-properties) to use "list of packages authors, matching the profile names on nuget.org." I've just haven't determined your NuGet account, @rspeele. If you haven't registered there, then we probably could just leave this as-is.